statsmodels/sandbox/distributions/tests/test_multivariate.py

```diff
@@ -63,15 +63,17 @@ def test_mvn_mvt_3(self):
         df = self.df
         corr2 = self.corr2
 
-        #from -inf
-        #print 'from -inf'
         a2 = a.copy()
         a2[:] = -np.inf
-        probmvn_R = 0.9961141 #using higher precision in R, error approx. 6.866163e-07
-        probmvt_R = 0.9522146 #using higher precision in R, error approx. 1.6e-07
-        assert_almost_equal(probmvt_R, mvstdtprob(a2, b, corr2, df), 4)
-        assert_almost_equal(probmvn_R, mvstdnormcdf(a2, b, corr2, maxpts=100000,
-                                                    abseps=1e-5), 4)
+        # using higher precision in R, error approx. 6.866163e-07
+        probmvn_R = 0.9961141
+        # using higher precision in R, error approx. 1.6e-07
+        probmvt_R = 0.9522146
+        quadkwds = {'epsabs': 1e-08}
+        probmvt = mvstdtprob(a2, b, corr2, df, quadkwds=quadkwds)
+        assert_almost_equal(probmvt_R, probmvt, 4)
+        probmvn = mvstdnormcdf(a2, b, corr2, maxpts=100000, abseps=1e-5)
```

Comment:
there is a typo in the existing code, the keyword is `epsabs` and not `abseps`

Reply:
No, it is `abseps`. 

https://github.com/scipy/scipy/blob/master/scipy/stats/mvndst.f#L35

`epsabs` is for quad.